libmscore/harmony.cpp

```diff
@@ -1059,10 +1059,15 @@ void Harmony::layout()
             xx += cw;
             xx -= width();
             }
-      else if ((align() & Align::HCENTER) || parent()->isFretDiagram()) {
+      else if (align() & Align::HCENTER) {
             xx += (cw * .5);
             xx -= (width() * .5);
             }
+      else if (parent()->isFretDiagram()) {
+            FretDiagram* fd = toFretDiagram(parent());
+            xx += fd->width() * .5;
```

Comment:
I guess the effect of this is to center the chord symbol even though it's set to left align.  Wouldn't it be better to actually coerce the alignment to center?  We play similar games with lyrics: "one" is centered, but in "one - - - - self" (melisma extended "one" over several notes) it gets left aligned.

Reply:
fixed by next commit